configs/README.md

```diff
@@ -0,0 +1,60 @@
+##  Config JSON Schema
+
+Benchmarks are configured by editing the config.json file.
+In the benchmarks settings can be configured: some algorithms parameters, datasets, a list of frameworks to use, and the usage of some environment variables.
+Below is a list and description of all fields in the configuration file.
+
+###  Root Config Object
+| Field Name  | Type | Description |
+| ----- | ---- |------------ |
+|omp_env| array[string] | for xgboost only. Specify an environment variable to set the number of omp threads |
+|common| [Common Object](#common-object)| **REQUIRED** common benchmarks setting: frameworks and input data settings |
```

Comment:
what about creating a separate column to mark whether or not the parameter is required?

Reply:
We discussed this. The problem is that we have some objects, for example `training` and `testing` objects, in which there are no required parameters. In this case, there will simply be an empty column that will look unaesthetically. To maintain overall consistency, we decided to simply move the word **Requred** to the start of the line.